packages/core/src/plugins/ventana/index.ts

```
@@ -0,0 +1,140 @@
+import { 
+    replaceObjectStaticMethods,
+    replaceObjectStaticProperties,
+    replaceObjects
+  } from '../utils'
+  export const report = () => 'Converts ventana to window'
+  
+  export const properties = new Map<string, string>([
+    ["caches", "caches"]
+    ["informacionCliente", "clientInformation"],
```

Biome

[error] The comma operator is disallowed.

Its use is often confusing and obscures side effects.

(lint/style/noCommaOperator)